src/style.css

```diff
@@ -1,3 +1,88 @@
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  --header-height: 60px;
+  --blue-color-hover: #00acdc;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+nav {
+  padding: 0;
+}
+
+.nav__list.nav__list:not(:last-child) {
+  margin-right: 20px;
+}
+
+@media (min-width: 1024px) and (max-width: 1199px) {
```

Comment:
No need to write media queries for this task. Use width: 100% for header

Reply:
didn't change it
https://github.com/mate-academy/layout_moyo-header/pull/3390/files#r1251857239

Reply:
i don't see any differences at 1024px to 1200px